va/va.h

```diff
@@ -492,6 +492,20 @@ typedef enum
      */
     VAConfigAttribEncMacroblockInfo     = 16,
     /**
+     * \brief Maximum picture width. Read-only.
+     *
+     * This attribute determines the maximum picture width the driver supports
+     * for a given configuration.
+     */
+    VAConfigAttribMaxPictureWidth     = 18,
+    /**
+     * \brief Maximum picture height. Read-only.
+     *
+     * This attribute determines the maximum picture height the driver supports
+     * for a given configuration.
+     */
+    VAConfigAttribMaxPictureHeight    = 19,
+    /**
```

Comment:
@xhaihao you mean use VASurfaceAttribMaxWidth VASurfaceAttribMaxHeight to report maximum resolution ?  I just wonder whether surface resolution is encode/decode resolution.  sometimes, customer create a big surface (Maybe for VP usage) but the container YUV to encoder is smaller.